variables.tf

```diff
@@ -23,16 +23,20 @@ variable "runtime" {
   type        = string
 }
 
+variable "source_archive" {
+  description = "(Required) Path to the '.zip' archive that contains the source code of this Cloud Function."
+}
+
+variable "bucket" {
+  description = "(Required) The URI of the bucket that the archive that contains the function and its dependencies will be uploaded to."
+  type        = string
+}
+
```

Comment:
hey @bartekrutkowski-ext .. those changes need to be reflected in the terradoc file and generated readme as the variables changed from optional to required.